container-app-jobs/blob-config-runner/start-jobs.py

```diff
@@ -0,0 +1,150 @@
+        entry[SELECTED] = selected
+
+# Collect all the selected items and return the config files names as a list, for processing
+def get_selected(config_files):
+    selected = []
+    for entry in config_files:
+        if(entry[SELECTED]):
+            selected.append(entry[NAME])
+
+    return selected
+
+# Load the config file
+config = configparser.ConfigParser()
+config.read('config.ini')
+
+# Create default credentials to give to the Azure SDK client. Used by both blob storage and container app jobs.
+credential = DefaultAzureCredential()
+
+# Create a blob service client object
+blob_service_client = BlobServiceClient(config.get('azure', 'account_url'), credential=credential)
+# Create a container client using the client for the container
+container_client = blob_service_client.get_container_client(container=config.get('azure', 'container_name'))
+# List the blobs in this container
+blob_list = container_client.list_blobs(name_starts_with=config.get('azure', 'prefix'))
+
+# Iterate over the items returned from blob storage and build a dictionary to keep track of their selection status
+config_files = []
+for blob in blob_list:
+    if blob.name.endswith('-config.json'):      # expected file naming to identify config files
+        config_files.append({NAME: blob.name, SELECTED: False})         # set each to not selected by default
+
+# Output some instructions
+print('Select config files to start jobs on. Select items by entering each index individually or as a comma separated list.')
+print('Additionally, enter \'A\' to select all, \'N\' to deselect all, or \'Q\' to quit.')
+print('When selections are complete, press Enter with no input to proceed.')
+
+# Print the dictionary
+print_configs(config_files)
+
+# Prompt the user for input in a loop, allowing them to select which config files they want to process
+val = '0'   # default value
+while val.upper() != '':        # loop until they enter nothing, which signifies continuing to the process step
+    val = input('Enter selection (or press enter to proceed): ')        # get user input
+
+    try:
+        index = int(val)        # attempt to convert the value to an integer
+        toggle_config(config_files, index)      # if successful, toggle that item
+    except ValueError:          # if int conversion failed, its not a number
+        if val.upper() == 'Q':
+            quit()      # quit the script
+        if val.upper() == 'A':
+            set_all(config_files, True)     # select all
+        elif val.upper() == 'N':
+            set_all(config_files, False)    # deselect all
+
+    if val != '':   # dont print the last time
+        # Print the dictionary
+        print_configs(config_files)
+
+# Collect the selected items and move on to processing step
+selected = get_selected(config_files)
+print(f'Processing {len(selected)} files: {selected}.')
+val = input('Confirm to proceed (Y)? ')     # confirm the list with the user
+
+if val.upper() == 'Y':
```

Comment:
I got tripped up here because I thought `(Y)` in the prompt implied an empty `<CR>` would be parsed as a `Y` the way it usually does with this type of prompt in the terminal. Maybe:

```suggestion
if val.upper() in ["Y", ""]:
```